getGit.tex

```diff
@@ -185,361 +215,201 @@ \section{What, Why, and How?}
-\end{verbatim}
+\begin{frame}
+    \vspace{1cm} % generate some space between title and content
+	\begin{columns}
+		\column{2.5in}
+    \frametitle{Our Sample Workflow}
+        1. Pull from Upstream Master to \\
+        Local Master \linebreak\linebreak
+        2. Branch to Working Directory \linebreak\linebreak
+        3. Make and test changes \linebreak\linebreak
+        4. Commit to Local Dev \linebreak\linebreak
+        5. Push to Origin Dev \linebreak\linebreak
+        6. Open pull request to Upstream \\
+        Master
+	\column{2.25in}
+	\begin{center}
+	\includegraphics[width = .9\linewidth]{assets/gitflow4}
+	\end{center}
+	\end{columns}
+    \vspace{1cm} % generate some space between title and content
 \end{frame}
 
-\begin{frame}[fragile]
-	\frametitle{Branch: development history}
-	When harvest time comes, you pick the good ones and leave the bad.
-\begin{verbatim}
-                   F
-                /--o
-               /
-A  B  C     E /  G  F      D  H  I  J(merged)
-o--o--o--\--o/---o--o-----/o--o--o--o
-          \              /
-           \--o--o--\--o/
-              D  H   \ I
-                      \--o
-                        DUMB
-\end{verbatim}
+\begin{frame}
+    \vspace{1cm} % generate some space between title and content
+	\begin{columns}
+		\column{2.5in}
+    \frametitle{Our Sample Workflow}
+        1. Pull from Upstream Master to \\
+        Local Master \linebreak\linebreak
+        2. Branch to Working Directory \linebreak\linebreak
+        3. Make and test changes \linebreak\linebreak
+        4. Commit to Local Dev \linebreak\linebreak
+        5. Push to Origin Dev \linebreak\linebreak
+        6. Open pull request to Upstream \\
+        Master
+	\column{2.25in}
+	\begin{center}
+	\includegraphics[width = .9\linewidth]{assets/gitflow5}
+	\end{center}
+	\end{columns}
+    \vspace{1cm} % generate some space between title and content
 \end{frame}
 
-\begin{frame}[fragile]
-	\frametitle{Branch: development history}
-	It's possible to clean up unused branches at any time.
-\begin{verbatim}
-                   F
-                /--o
-               /
-A  B  C     E /  G  F      D  H  I  J(merged)
-o--o--o--\--o/---o--o-----/o--o--o--o
-          \              /
-           \--o--o--\--o/
-              D  H   \ I
-                      \--o
-                        DUMB
-\end{verbatim}
+\begin{frame}
+    \vspace{1.01cm} % generate some space between title and content
+	\begin{columns}
+		\column{2.5in}
+    \frametitle{Our Sample Workflow}
+        1. Pull from Upstream Master to \\
+        Local Master \linebreak\linebreak
+        2. Branch to Working Directory \linebreak\linebreak
+        3. Make and test changes \linebreak\linebreak
+        4. Commit to Local Dev \linebreak\linebreak
+        5. Push to Origin Dev \linebreak\linebreak
+        6. Open pull request to Upstream \\
+        Master
+	\column{2.25in}
+	\begin{center}
+	\includegraphics[width = .95\linewidth]{assets/gitflow6}
```

Comment:
This slide's animation doesn't register well with the prior slides. Why did you have to adjust the `vspace` and `includegraphics` scales?

Reply:
Thanks for raising an issue about this, @cartertaylor.